app/views/layouts/react_page.haml

```diff
@@ -1,4 +1,108 @@
-= render :partial => 'layouts/head'
+!!! 5
+
+%head
+  %meta{charset: "utf-8"}
+
+  -# Kissmetrics is deprecated, use Google Tag Manager instead
+  = render :partial => "layouts/kissmetrics"
```

Comment:
MikkoP, but reading from the original commit, it looks pretty clear that this is just graceful deprication (for OS, I guess). I'll remove